src/test/java/FilterAndSearch.java

```diff
@@ -0,0 +1,98 @@
+        String brandName = brandLi.getAttribute("aria-label").toLowerCase();
+        WebElement brandInput = brandLi.findElement(By.tagName("input"));
+        brandInput.sendKeys(Keys.SPACE);
+        webDriver.manage().timeouts().implicitlyWait(Duration.ofSeconds(2));
+
+        List<WebElement> searchResults = webDriver.findElements(By.xpath("//*[@data-component-type=\"s-search-result\"]"));
+        for (WebElement element : searchResults) {
+            WebElement h2 = element.findElement(By.tagName("h2"));
+            assert h2.getText().toLowerCase().startsWith(brandName);
+        }
+    }
+
+    @Test
+    public void PriceVerify() {
+        WebElement lowPrice = webDriver.findElement(By.xpath("//*[@id=\"low-price\"]"));
+        lowPrice.sendKeys("100");
+        WebElement highPrice = webDriver.findElement(By.xpath("//*[@id=\"high-price\"]"));
+        highPrice.sendKeys("150");
+        WebElement submitButton = webDriver.findElement(By.xpath("//*[@id=\"priceRefinements\"]//form"));
+        submitButton.submit();
+        webDriver.manage().timeouts().implicitlyWait(Duration.ofSeconds(2));
+
+        List<WebElement> searchResults = webDriver.findElements(By.xpath("//*[@data-component-type=\"s-search-result\"]"));
+        for (WebElement element : searchResults) {
+            try {
+                WebElement whole = element.findElement(By.className("a-price-whole"));
+                WebElement fraction = element.findElement(By.className("a-price-fraction"));
+                String result = whole.getText() + "." + fraction.getText();
+                double price = Double.parseDouble(result);
+                assert price >= 100.00;
+                assert price <= 150.00;
+            } catch (Exception e) {
+            }
+        }
+    }
+
+    @Test
+    public void PriceSorting() {
+        WebElement DropdownPrompt = new WebDriverWait(webDriver, Duration.ofSeconds(10))
+                .until(ExpectedConditions.visibilityOfElementLocated(By.xpath("//*[@class=\"a-dropdown-prompt\"]")));
+        DropdownPrompt.click();
+        WebElement lowToHigh = webDriver.findElement(By.xpath("//*[@id=\"s-result-sort-select_1\"]"));
+        lowToHigh.click();
+        webDriver.manage().timeouts().implicitlyWait(Duration.ofSeconds(2));
+
+        List<WebElement> searchResults = webDriver.findElements(By.xpath("//*[@data-component-type=\"s-search-result\"]"));
+        double price = 0.00;
+        for (WebElement element : searchResults) {
+            try {
+                WebElement whole = element.findElement(By.className("a-price-whole"));
+                WebElement fraction = element.findElement(By.className("a-price-fraction"));
+                String result = whole.getText() + "." + fraction.getText();
+                double currentPrice = Double.parseDouble(result);
+                assert currentPrice >= price;
+                price = currentPrice;
+            } catch (Exception e) {
+            }
```

Comment:
If you use try-catch, please put some logic in a catch block. 
And it is okay that your tests failed, I know that functionality can work incorrectly, so it is better to remove this try-catch block